mobile/screens/tab_navigator/indica_ai/containers/ListLocals.js

```diff
@@ -60,11 +62,19 @@ class ListLocals extends Component {
           return (
             <ScrollView>
               {locals.map( local =>
+                <TouchableHighlight
```

Comment:
O esperado é que essa lógica esteja totalmente dentro do componente local, e não no container ListLocals. Assim, sempre que um Local for renderizado a gente vai conseguir ir para a página dele. Da forma que está, seria necessário replicar a lógica de navegação toda vez que o componente Local for renderizado.